sci-chemistry/modeller/modeller-10.5.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
Please don't do "See ...", at least not by itself. The way we do "See" is a `Bug:` tag (which means tag the bug but don't close it).

Reply:
Yep, you're right. Fixed.